tools/experiments/experiments.html

```diff
@@ -40,6 +40,10 @@
       color: #147bc1;
     }
 
+    .version {
+      color: #3ab3eb;
```

Comment:
This makes it look like a link. Consider a more neutral colour?

Reply:
Reformatted to fixed-width / gray.